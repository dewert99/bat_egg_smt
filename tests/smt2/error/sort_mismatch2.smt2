(declare-sort S1)
(declare-fun f (Bool) S1)
(assert (f true))