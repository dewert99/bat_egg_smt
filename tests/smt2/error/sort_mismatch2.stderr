the 0th argument of the function assert has sort S1 but should have sort Bool at 3:12
(assert (f true))
           ^

